backend/src/main/java/com/sim_backend/websockets/messages/HeartBeat.java

```diff
@@ -1,9 +1,8 @@
 package com.sim_backend.websockets.messages;
 
-import com.sim_backend.websockets.OCPPMessage;
 import com.sim_backend.websockets.OCPPMessageInfo;
 
-@OCPPMessageInfo(messageName = "HeartBeat")
+@OCPPMessageInfo(messageType = "HeartBeat", messageName = "HeartBeat")
```

Comment:
Some confusion regarding messageType and messageName as they were being used interchangeably in some places. For now, I have both being required as part of the OCPPMessage but I think both are not required.